utility/utils.py

```diff
@@ -635,6 +635,8 @@ def run_cmd(cmd, **kwargs):
 
     Returns:
         (str) Decoded stdout of command
+        (str) Decoded stderr of command
```

Comment:
You want to convert this to list here.

Reply:
Would it not be a tuple instead?

Ex.
```
(str, str, str) Decoded stdout, decoded stderr, and return code of command
```

Reply:
yeah either list or tuple, the return signature should change, tuple is more valid here.

Reply:
Done. Changed the return type to tuple.